components/ClaimDetails.tsx

```diff
@@ -61,7 +54,11 @@ export const ClaimDetails: React.FC<ClaimDetailsProps> = ({
 
             {weeklyBenefitAmountField}
 
-            {lastPaymentIssuedField}
+            <InfoField
+              loading={loading}
+              label={t('claim-details.last-payment-issued')}
+              text={lastPaymentIssued || t('claim-details.none')}
```

Comment:
This is built on top of existing tested logic that sets this field to null in either case.

Logic:
https://github.com/cagov/ui-claim-tracker/blob/d0ac954ace870af5b203f2708c5bc6da10bda166/utils/getClaimDetails.ts#L176-L179

Tests:
https://github.com/cagov/ui-claim-tracker/blob/d0ac954ace870af5b203f2708c5bc6da10bda166/tests/utils/getClaimDetails.test.ts#L176-L226

Reply:
also we set it here instead of back there because _this_ is the layer of our code where we resolve our translations